tizen/src/fltypes.h

```diff
@@ -0,0 +1,11 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_FLTYPES_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_FLTYPES_H_
+
+#include <flutter/method_call.h>
+#include <flutter/method_channel.h>
+
+using FlMethodCall = flutter::MethodCall<flutter::EncodableValue>;
+using FlMethodResultRef =
+    std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>>;
```

Comment:
Rather than separating it into separate header files and including it, I think it would be better to use include and declaration directly where it is used.

Reply:
But what if I use those definition in more than 2 files, is declaring directly still a good approach?

Reply:
In the part used for readability, the relevant header file is included, and `typedef` is also intended to be specified in that file. Therefore, I think it is unnecessary to separate the simple declaration into a separate header file.
For similar information, it would be good to refer to the implementation of `tizen_rpc_port` of flutter-tizen/plugins.

Reply:
Thx! I'll check for it.

Reply:
Removed `fltypes.h`.